lynx/src/main/java/com/github/pedrovgs/lynx/LynxConfig.java

```diff
@@ -55,10 +58,21 @@ public LynxConfig setMaxNumberOfTracesToShow(int maxNumberOfTracesToShow) {
   }
 
   public LynxConfig setFilter(String filter) {
+    if (filter == null) {
+      throw new NullPointerException("filter can't be null");
```

Comment:
I'm using IllegalArgumentException for this kind of guards. What do you think about this other implementation?

```java

public LynxConfig setFilter(String filter) {
    if(filter == null){
        throw new IllegalArgumentException("You can't use a null instance as filter");
    }
}

```